openeo/testing/results.py

```diff
@@ -194,10 +196,11 @@ def _compare_xarray_dataarray(
     *,
     rtol: float = _DEFAULT_RTOL,
     atol: float = _DEFAULT_ATOL,
+    pixel_tolerance: float = _DEFAULT_PIXELTOL,
     name: str = None,
 ) -> List[str]:
     """
-    Compare two xarray DataArrays with tolerance and report mismatch issues (as strings)
+    Compare two xarray DataArrays either with tolerance or with allowable pixels and report mismatch issues (as strings)
```

Comment:
This doc change is now not necessary anymore I think, as we can keep it general in terms of "tolerance"
```suggestion
    Compare two xarray DataArrays with tolerance and report mismatch issues (as strings)
```